index.ts

```diff
@@ -155,8 +120,6 @@ async function init() {
 
 	updateStatus('Retrieving directory info');
 
-	const {private: repoIsPrivate} = await fetchRepoInfo(`${user}/${repository}`);
```

Comment:
This was never reached because "parseUrl" would just output an error and end up on "Unknown error"